Cargo.toml

```diff
@@ -32,3 +33,12 @@ lto = "thin"
 incremental = true
 debug-assertions = true
 debug = true
+
+# Uncomment these lines for local development paths
```

Comment:
Well, we put the URLs in each Cargo.toml of the project, so we can as well delete this.
Weren't you able to find any other solution ?

Reply:
Actually what you see in Cargo.toml now - is the only "solution" I found for more effective development. I am uncommenting these lines in all repos, and local folders instead of GIT urls are used. In case of mixing local paths and GIT urls I am getting the incompatibility error in all combinations.

Reply:
Understood :/

Reply:
Maybe add some instructions on the readme.MD to explain this and open an issue